src/chronos/chronos2/config.py

```diff
@@ -83,11 +86,16 @@ def __init__(
 
         assert not self.is_gated_act, "gated activation is not supported"
 
+        # Attention implementation - default to "sdpa" if not specified
+        attn_implementation = attn_implementation or "sdpa"
```

Comment:
Could we just set the default value in the function signature to `"sdpa"`, or is there some case where the current logic is required?